core/story-api/src/main/java/org/lgna/story/implementation/StoryApiDirectoryUtilities.java

```diff
@@ -131,30 +132,24 @@ private static void askUserForModelGallery() {
   }
 
   public static File getSoundGalleryDirectory() {
-    try {
-      File installDirectory = getInstallDirectory();
-      if (installDirectory != null) {
-        File soundGalleryDirectory = new File(installDirectory, SOUND_GALLERY_NAME);
-        if (soundGalleryDirectory.isDirectory()) {
-          return soundGalleryDirectory;
-        } else {
-          throw new RuntimeException(); //fallback
-        }
-      } else {
-        throw new NullPointerException(); //fallback
-      }
-    } catch (Throwable t) {
-      return getFallbackDirectory();
-    }
+    return getDirectory(SOUND_GALLERY_NAME);
   }
 
   public static File getStarterProjectsDirectory() {
+    return getDirectory(STARTER_PROJECTS_NAME);
+  }
+
+  public static File getInternalModelsDirectory() {
+    return getDirectory(INTERNAL_MODELS_NAME);
+  }
+
+  private static File getDirectory(String name) {
     try {
       File installDirectory = getInstallDirectory();
       if (installDirectory != null) {
-        File starterProjectsDirectory = new File(installDirectory, STARTER_PROJECTS_NAME);
-        if (starterProjectsDirectory.isDirectory()) {
-          return starterProjectsDirectory;
+        File soundGalleryDirectory = new File(installDirectory, name);
```

Comment:
soundGalleryDirectory looks like it snuck through here, swap this to a more generic version.

this is much cleaner now!